change/@internal-react-composites-093c2064-8b62-442d-bf87-d075572716be.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Update styles around split button border radius",
```

Comment:
Try be more specific here; `Increase control bar button border radius in Composites with Call features`